bruno/tickets/get-ticket-by-id.bru

```diff
@@ -1,11 +1,11 @@
 meta {
   name: get-ticket-by-id
   type: http
-  seq: 2
+  seq: 4
 }
 
 get {
-  url: http://localhost:3000/tickets/4
+  url: http://localhost:3000/tickets/8
```

Comment:
change to use variables for the environment if the variable is dynamic use a secret variable